standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -7502,6 +7502,15 @@ public boolean delete_column_statistics_req(DeleteColumnStatisticsRequest req) t
           }
         }
       }
+      // on the table level, partially delete(update) table level parameter COLUMN_STATS_ACCURATE
+      if (colNames == null || colNames.isEmpty()){
+        // remove all column names in parameter COLUMN_STATS_ACCURATE
+        StatsSetupConst.clearColumnStatsState(table.getParameters());
+      } else {
+        // remove the deleted column names in parameter COLUMN_STATS_ACCURATE
+        StatsSetupConst.removeColumnStatsState(table.getParameters(), colNames);
+      }
+      rawStore.alterTable(parsedDbName[CAT_NAME], parsedDbName[DB_NAME], tableName, table, null);
```

Comment:
I don't like the idea of doing an alter operation just to update 'COLUMN_STATS_ACCURATE' field.

Reply:
we might need to add the new interface in RawStore, example `void updateTableParam(long tableId, String paramKey, String newParamValue)` to update the table's parameters